server/lib/data_import/my_anime_list/extractor/media.rb

```diff
@@ -0,0 +1,139 @@
+class DataImport::MyAnimeList
+  module Extractor
+    class Media
+      attr_reader :data
+
+      def initialize(json)
+        @data = JSON.parse(json)
+      end
+
+      def age_rating
+        rating = data['classification'].split(' - ')[0]
+
+        case rating
+        when 'G','TV-Y7' then :G
+        when 'PG', 'PG13' then :PG
+        when 'R', 'R+' then :R
+        when 'Rx' then :R18
+        end
+      end
+
+      def episode_count
+        data['episodes']
+      end
+
+      def episode_length
+        data['duration']
+      end
+
+      def synopsis
+        clean_desc(data['synopsis'])
+      end
+
+      def youtube_video_id
+        data['preview']
+      end
+
+      def poster_image
+        data['image_url']
+      end
+
+      def age_rating_guide
+        rating = data['classification'].split(' - ')[0]
```

Comment:
ok, so all of them will return perfectly except for `R`, it returns `17+ (violence & profanity)`, instead of just `Violence, Profanity` (like we have now). Is that okay?